assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm

```diff
@@ -995,6 +995,106 @@
                         </div>
 
                         <!-- End VeriCite custom options -->
+					#elseif($reviewServiceProviderId == 1571541951)
```

Comment:
This value also in AssignmentConstants, are these never going to change?

Reply:
These values will never change. They are the hash IDs for "Turnitin" and "TurnitinOC". Since the content review service uses the display name for both the unique ID and in the UI for display, I had to distinguish the two "Turnitin" implementations by their Hash ID (which is also how the DB distinguishes them). I didn't think they need to be added to AssignmentConstants since the others are not ("vericite", "urkurd", "turnitin")

Reply:
I still feel like this numeric  value shouldn't be in the assignment.vm, if this is every changed nobody will know what this means. It should either be pulled out to the assignment constants file or maybe the ContentReviewConstants would make more sense, since as you say there's no other constants for Urkund, etc in AssignmentConstants but there is in ContentReviewConstants